tests/specs/benchmarks/verification.k

```diff
@@ -26,6 +26,9 @@ module VERIFICATION
     imports VERIFICATION-COMMON
     imports VERIFICATION-JAVA
 
+    rule ( M:Map [ K1 <- _  ] ) [ K2 <- V2 ] =>   M [ K1 <- V2 ]                requires K1 ==Int K2
+    rule ( M:Map [ K1 <- V1 ] ) [ K2 <- V2 ] => ( M [ K2 <- V2 ] ) [ K1 <- V1 ] requires K1  >Int K2
```

Comment:
```suggestion
    rule ( M:Map [ K1 <- V1 ] ) [ K2 <- V2 ] => ( M [ K2 <- V2 ] ) [ K1 <- V1 ] requires K2  <Int K1
```

Let's have all new added lemmas use `<Int` and `<=Int` instead (since it reads lowest to highest that way).
I also don't think you need the paranthesis.

Reply:
done